docker/android-system.sh

```diff
@@ -8,12 +8,21 @@ set -euo pipefail
 
 main() {
     local arch="${1}"
+    local version="${ANDROID_VERSION}"
+    local gcc_version=4.9
+    local glibc_version=glibc2.15-4.8
```

Comment:
are these guaranteed for future versions?

Reply:
I don't believe so. The GCC version likely is, but there's multiple glibc versions present, which possibly could be removed at some point in the future. Unlikely, but possible. I want to be able to version-gate it. We also might want to upgrade it at the very least: some newer versions include `prebuilts/gcc/linux-x86/host/x86_64-linux-glibc2.19-4.9`.

Reply:
Actually this is Android version-dependent. I keep forgot the platform meta repository lists repositories that have been removed. Updated to:

```bash
    case "${major_version}" in
        4)
            glibc_version=glibc2.7-4.6
            ;;
        5|6|7|8|9)
            glibc_version=glibc2.15-4.8
            ;;
        10|11|12|13)
            glibc_version=glibc2.17-4.8
            ;;
        *)
            # current master branch as of 9/15/2022
            glibc_version=glibc2.19-4.9
            ;;
    esac
```